cypress/page/complete-checkout.page.ts

```diff
@@ -3,11 +3,11 @@ class FinishCheckout{
     private messageElement: string;
 
     constructor() {
-        this.messageElement = "#contents_wrapper > .checkout_complete_container > h2";
+        this.messageElement = "Thank you for your order!";
```

Comment:
The way you have it was correct!, because you need to validate that an element contains a specific text. An idea to improve this selector is to reduce it in this way: 

constructor() {
    this.messageElement = '#checkout_complete_container h2';
  }

Reply:
Updated the element selector in the second commit. Please review